libs/market-depth/src/lib/orderbook.spec.tsx

```diff
@@ -183,4 +183,53 @@ describe('Orderbook', () => {
     );
     expect(result.getByTestId('scroll').scrollTop).toBe(6 * rowHeight);
   });
+
+  it('should format correctly the numbers on resolution change', async () => {
+    const onClickSpy = jest.fn();
+    window.innerHeight = 11 * rowHeight;
+    const result = render(
+      <Orderbook
+        decimalPlaces={decimalPlaces}
+        positionDecimalPlaces={0}
+        onClick={onClickSpy}
+        fillGaps
+        {...generateMockData(params)}
+        onResolutionChange={onResolutionChange}
+      />
+    );
+    await waitFor(() => screen.getByTestId(`bid-vol-${params.midPrice}`));
```

Comment:
suggestion (non blocking): this would be better as 
```
expect(await screen.findByTestId(`bid-vol-${params.midPrice}`)).toBeInTheDocument()
```

Reply:
updated